components/Connect.tsx

```diff
@@ -70,6 +70,46 @@ export default function BluetoothConnect({ bikeCredentials, setBikeInstance, bac
                     Back to login
                 </Button>
             </P>
+
+            <P vertical={10}>
+                <Button
+                    onClick={() => {
+                        const blob = new Blob([JSON.stringify(bikeCredentials)], { type: 'application/json' })
```

Comment:
A fancy json file is much better for these kinds of things.
_Adding `, null, 4` to a `JSON.stringify` will print a fancy JSON file_
```suggestion
                        const blob = new Blob([JSON.stringify(bikeCredentials, null, 4)], { type: 'application/json' })
```